### ISSUE TITLE
RTF text area type is obsolete

### ISSUE THREAD

**Issue by christillman** [christillman/encounterpro_os#18]
### Contact Details

toff.tillman@gmail.com

### What happened?

I was looking into why the display scripting is so slow. As part of that, I discovered that the text area type we have enabled for the application is obsolete. 

![image](https://github.com/christillman/encounterpro_os/assets/25717203/f36650f6-36d1-44ba-87ce-5e5b5ad6fff6)

I have selected the recommended version now, to be applied in release 7.2.1.3. We need to continue using 32-bit compilation for now, in order to support the EPro utilities and prevent random breakage. But the same text area type is also available for 64-bit if we need to go that way eventually.

![image](https://github.com/christillman/encounterpro_os/assets/25717203/7eaaf5d1-cf86-4eaa-9801-e536b61bca02)

### How did it happen?

Investigatiing text area usage to improve performance. Unfortunately selecting the recommended type doesn't seem to affect performance either positively or negatively. Everything still seems to work the same. But it will be less prone to bugs and more well supported going forward. I read that the company that supplies the text area to Appeon has signed an agreement in late 2022 to extend their partnership.

### Expectations

I expected ...

### Urgency and Importance

I need ...

### Version

other

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by christillman** [comment]
As to the performance, I added some intrumentation to record how long the display script takes and which parts take longer than others. After getting a baseline, I tried making the text area invisible while it was being assembled. This improved the performance on the one I was testing from 70 seconds to 40 seconds. I needed to add a progress indication while the text area was hidden, or users would give up, so I did that as well. It's ready for the next release (7.2.1.3). I attached the performance summary I did.
